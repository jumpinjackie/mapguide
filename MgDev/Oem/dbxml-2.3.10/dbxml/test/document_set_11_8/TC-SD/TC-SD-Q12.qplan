<XQuery>
  <FLWOR>
    <ForBinding name="a">
      <DbXmlNav>
        <QueryPlanFunction result="collection" container="11.8.4.dbxml">
          <OQPlan>V(node-element-equality-string,hw,=,'his')</OQPlan>
        </QueryPlanFunction>
        <DbXmlStep axis="child" name="dictionary" nodeType="element"/>
        <DbXmlStep axis="child" name="e" nodeType="element"/>
        <DbXmlFilter>
          <DbXmlNav>
            <DbXmlStep axis="child" name="hwg" nodeType="element"/>
            <DbXmlCompare name="equal" join="child" name="hw" nodeType="element">
              <Sequence>
                <AnyAtomicTypeConstructor value="his" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
              </Sequence>
            </DbXmlCompare>
          </DbXmlNav>
        </DbXmlFilter>
      </DbXmlNav>
    </ForBinding>
    <DOMConstructor type="element">
      <Name>
        <Sequence>
          <AnyAtomicTypeConstructor value="Entry" typeuri="http://www.w3.org/2001/XMLSchema" typename="QName"/>
        </Sequence>
      </Name>
      <Children>
        <DbXmlNav>
          <Variable name="a"/>
          <DbXmlStep axis="child" name="ss" nodeType="element"/>
        </DbXmlNav>
      </Children>
    </DOMConstructor>
  </FLWOR>
</XQuery>
