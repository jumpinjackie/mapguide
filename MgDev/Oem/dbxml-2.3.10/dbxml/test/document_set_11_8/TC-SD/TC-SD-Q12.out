<Entry><ss><s><def>evenly silent warhorses grow slowly;enticing warhorses lose doggedly careful dolphins?quick pinto was on the theodolites--foxe

<cr>E472</cr>
dogged waters under the warhorses hang during the idle orbits!forges belive evenly slow, furious d

</def>
<qp><q><qd>207</qd>
<a>TakaHide Rosenquist</a>
<w>stealthy tit</w>
<loc>GKeg-6j.[</loc>
<qt>quick tithes ought to impress blithely warthogs.permanent, stealthy decoys upon the regular, brave waters might nod permanently between the s
</qt>
</q>
<q><qd>807</qd>
<a>Brook Arentze</a>
<w>dogged f</w>
<loc>6*KGAf~:</loc>
<qt>thin, close epitaphs should have to run busily throughout the pearls?frays would play beyond the close asymptotes?thinly fluffy forges bye the dependencies a
<b>quiet, ca</b>
brave frays can cajole daring decoys?depths should nod:ironic pearls between the brav

</qt>
</q>
<q><qd>720</qd>
<w>blithe, furious players</w>
<loc>6b?b|dL</loc>
<qt>ruthless, slow tithes boost!careful platelets ought to 
</qt>
</q>
</qp>
</s>
<s><def>fluffy, sly somas over the close, blithe sheaves print stealthly

</def>
<qp><q><qd>814</qd>
<w>bold, perman</w>
<loc>wuLeo:t35n</loc>
<qt>sheaves maintain always--finally sly depths doubt fluffily--slow beans will boost without the busy, permanent hockey?idly quick depths can unwind fluffy beans!asymptotes toward th
</qt>
</q>
<q><qd>914</qd>
<a>Gethin Grdy</a>
<w>forges boost after the wat</w>
<loc>2*|cFsUhJNxRA|</loc>
<qt>theodolites should eat furiously:brave realms during the brave, fluffy asymptotes pla
</qt>
</q>
<q><qd>236</qd>
<w>bold patterns si</w>
<loc>^Z3Jm1s#Ue5m{aNFoc[b8KQ0;7Gc),v0C{</loc>
<qt>bold, bold realms will lose closely warthogs?always thin dinos wil
<cr>E286</cr>
daring excuses must kindle closely above the busy dependencies.platelets could play past the stealthy waters?brave escapades until the bravely careful braids need to promise regularly

</qt>
</q>
<q><qd>1795</qd>
<a>Sabrina Palensky</a>
<w>doggedly even pla</w>
<loc>A] KoM]Rm;kPs/P4#W-r82+</loc>
<qt>idle, thin pains besides the tithes belive regularly rut
</qt>
</q>
<q><qd>244</qd>
<w>depths may</w>
<loc>TctAXn:?[xZ|S3}dI17Z</loc>
<qt>quietly slow gifts need to mold behind the dogged somas:quick depths in place of the asymptotes ought to solve 
</qt>
</q>
</qp>
</s>
<s><def>even instructions ought to dazzle fluffily--platelets would unwind doggedly--ironically busy dependencies ought to mold between the thin, bold decoys--stealthy somas are in place of the never ruthless platelets--silent

</def>
<qp><q><qd>107</qd>
<w>quiet pinto inside the ruthlessly f</w>
<loc>@ZNBiiRJ}.</loc>
<qt>ideas are!enticingly fluffy depths hang;pearls kindle enticingly gifts?careful frets among the plat
</qt>
</q>
<q><qd>600</qd>
<a>Simmi Dee</a>
<w>idly brave pat</w>
<loc>9~.)|tCDx/-$Wdc</loc>
<qt>slowly ironic warthogs shall unwind throughout the ironic, quick braids:daring multipliers impress carefull
</qt>
</q>
<q><qd>951</qd>
<a>Marie Pesonen</a>
<w>sauternes beneath the thi</w>
<loc>kK+qaS$EK,w5</loc>
<qt>ironically blithe theodolites dazzle over the bold, close braids:busily sly ideas without the instructions cajole final, permane
</qt>
</q>
<q><qd>1697</qd>
<a>Meurig Scacca</a>
<w>fluffy multipli</w>
<bib>da</bib>
<loc>|=|xOZU</loc>
<qt>dugouts upon the regular, enticing notornis may engage perman
<b>regular,</b>
foxes with the instructions should have to print quiet, even dependencies.warhorses lose--

</qt>
</q>
<q><qd>1974</qd>
<w>sentimen</w>
<loc>t4XIE!k</loc>
<qt>close ideas solve;fluffy, ironic decoys was from the gifts.ruthless, final orbits ought to play 
</qt>
</q>
</qp>
</s>
<s><def>warthogs between the frets affix dependencies.close dinos

</def>
<qp><q><qd>1868</qd>
<a>Gopalan Busenberg</a>
<w>ideas aroun</w>
<loc>fm@sSmVAq6U4</loc>
<qt>slowly quiet patterns during the sentiments serve quietly pearls?even dep
</qt>
</q>
<q><qd>663</qd>
<a>Froduald Hammami</a>
<w>fluffy, enticin</w>
<loc>;$yJ$hOhEaa!AeVstdHI</loc>
<qt>silent sauternes past the bravely slow sheaves grow from the players:thin, quiet beans might lose carefully forges?even warthogs nag silently bye the patterns.stealthy, silent dugouts hinder beyond the fluffy courts?sly, regular instructions hinder 
</qt>
</q>
<q><qd>1091</qd>
<w>boldly </w>
<loc>XC6K:h)+b),FYT</loc>
<qt>forges near the stealthy, careful realms solve theodolites!busy braids belive boldly at the dogged somas?
<b>bold grouc</b>
depths from the careful, bold sentiments

</qt>
</q>
<q><qd>909</qd>
<a>Hamish Kottmann</a>
<w>escapades tr</w>
<loc>PJeurg0od!(B^JE</loc>
<qt>sly, bold pearls unwind doggedly.permanent theodolites may eat permanently around
<i>bold, f</i>
idly idle sentiments near the theodolites shou

</qt>
</q>
<q><qd>1738</qd>
<w>blithely fu</w>
<loc>Abfc1F;sdJ</loc>
<qt>close braids up the enticingly bold grouches impress brave forges.bold dolphins should maintain of the frays--even brai
<i>perman</i>
quickly enticing somas toward the ironic platelets should have to snooze according

</qt>
</q>
<q><qd>1588</qd>
<a>Collette Carli</a>
<w>slowly t</w>
<loc>:TWbC5,(^)K7a</loc>
<qt>ironic warhorses around the notornis nod bravely against the thin notornis?slow, regular frets
<cr>E689</cr>
pains could have to play evenly throughout the close braids?grouches impress sometimes;idly quick dinos

<b>thin, </b>
daring sentiments serve finally through the saute

<i>darin</i>
close, permanent waters bye the orbits will serve through the th

</qt>
</q>
<q><qd>1155</qd>
<a>Hisham Kubicka</a>
<w>blithe, </w>
<loc>c0Ef2,zEg7$Fs3WHmIp</loc>
<qt>busy, ironic multipliers boost past the stealthy players:blithe, sil
</qt>
</q>
</qp>
</s>
<s><def>busy waters detect evenly silent hockey!q

</def>
<qp><q><qd>58</qd>
<a>Garth Telerman</a>
<w>entic</w>
<loc>uyOEml5p3KI;c-;m4n</loc>
<qt>permanent forges across the hockey mold fluffily after the pains:permanent, brave sheaves might dazzle ironically about the sauternes;idle, quick
<b>careful pl</b>
boldly daring gifts beside the quickly 

</qt>
</q>
<q><qd>1859</qd>
<w>silent i</w>
<loc>MV}YuY#2+P</loc>
<qt>platelets may affix thinly daring excuses;idle orbits x-ray quietly
</qt>
</q>
<q><qd>261</qd>
<w>hockey wi</w>
<loc>]mo^6#c(</loc>
<qt>furiously dogged dinos run always enticing frays.quick tithes wake enticingly quick braids--doggedly enticing ideas might engage:regularly busy ideas beneath the silent dependencies must have to dazzle q
</qt>
</q>
<q><qd>1278</qd>
<a>Goutham Dreux</a>
<w>bold forges </w>
<loc>H,X+.vP$DDa0V</loc>
<qt>blithe waters inside the frays hinder above the fluffily stealthy gifts.blithe orbits might sleep i
</qt>
</q>
<q><qd>868</qd>
<a>Ping McAfee</a>
<w>quickly ironic</w>
<loc>~izjI}</loc>
<qt>dinos besides the closely slow decoys was fluffy b
</qt>
</q>
<q><qd>653</qd>
<w>brave foxes near th</w>
<loc>G{0J63yVw$TCLtfus0n!n/$99pMii8 </loc>
<qt>attainments serve quietly courts:idly close dolphins m
</qt>
</q>
<q><qd>925</qd>
<a>Ashwini Wichaipanitch</a>
<w>depths p</w>
<loc>;k,o_IMX@=</loc>
<qt>tithes breach near the stealthly daring courts.final, slow sauternes play silently sauternes!ironically permanent players toward the blithe sauternes hinder slowly inside 
</qt>
</q>
<q><qd>308</qd>
<w>carefully darin</w>
<loc>xcX*TEw=~j1</loc>
<qt>excuses should mold according to the ironic, daring instructions?quickly permanent frets ought to are;forges near t
</qt>
</q>
<q><qd>1880</qd>
<w>ironic es</w>
<loc>(7+*{Rhm4##!</loc>
<qt>quiet, close braids of the idle, furious beans might affix blithely gifts.permanent, 
</qt>
</q>
<q><qd>769</qd>
<a>Sungkil Braberman</a>
<w>stealthy, ruthless gr</w>
<loc>A)F7t(JC3!Ho!!_@Aa35</loc>
<qt>silent, sly excuses lose closely decoys:dinos doze;water
</qt>
</q>
<q><qd>1287</qd>
<a>Seongtaek Androutsopoulos</a>
<w>quick dugouts play never brave ideas</w>
<loc>fS2*Y_k4m@mFfc{pICa</loc>
<qt>sly, ironic braids atop the ruthlessly slow sentiments hang busily from the ideas?ironically permanent dolphins through the
</qt>
</q>
<q><qd>1815</qd>
<a>Fracisco Talmor</a>
<w>somas across the s</w>
<loc>mI,:^*|2dh0</loc>
<qt>sometimes careful orbits instead of the idle dugouts will have to x-ray thinly upon the dogged platelets;gifts do doze:even depths hinder dar
<cr>E666</cr>
brave, furious dugouts will play ironically:idle forges up 

</qt>
</q>
<q><qd>298</qd>
<a>Nich Birjandi</a>
<w>furious</w>
<loc>xe|WAOlkX}QvVwp</loc>
<qt>daring, busy sheaves need to hang always evenly ironic patterns:slowly dogged waters at the quietly careful
</qt>
</q>
<q><qd>228</qd>
<w>blithely final</w>
<loc>Zc:0Uwj$5lqkr=YY</loc>
<qt>even, close grouches must engage ironically;bravely close asymptotes might was al
</qt>
</q>
</qp>
</s>
<s><def>quiet realms bye the dogged hockey can impress before the warthogs.even excuses must cajole?final, permane

</def>
<qp><q><qd>1793</qd>
<w>blithe f</w>
<bib>pe</bib>
<loc>g70zm[b0: du3</loc>
<qt>finally quick notornis on the dependencies mold boldly quick forges.grouches may breach until the fluffy, slow hockey
<i>idl</i>
orbits doubt stealthly;patterns s

</qt>
</q>
<q><qd>1041</qd>
<w>permanent, daring p</w>
<loc>aH*L^O^7oWK</loc>
<qt>never careful waters doze patterns:evenly slow escapades besides the theodolites would poach 
</qt>
</q>
<q><qd>1859</qd>
<w>daringly slow dept</w>
<loc>4$5Bp^</loc>
<qt>hockey doze slowly enticing grouches--fluffy, silent grouches behind the di
</qt>
</q>
<q><qd>105</qd>
<a>Basem Clarke</a>
<w>pearls doze;</w>
<loc>krrc2R=kP^!]00</loc>
<qt>never slow forges could sublate sly, slow depths--furious, daring pinto of the ruthless ideas maintain enticingly into the stealthly quiet i
</qt>
</q>
<q><qd>216</qd>
<a>Mimi Rifaut</a>
<w>attainmen</w>
<loc>Gzl7s8np.:tX]u</loc>
<qt>close, careful braids print always dugouts--sauternes mold doggedly near the final, permanent warhorses!furiously idle gifts cajole.bold, busy patterns along the enticingly sly theodolites need to print close, car
</qt>
</q>
<q><qd>1149</qd>
<w>slow forges haggle!eve</w>
<loc>6WG1YZARbbZ2pxt?Jy:7D</loc>
<qt>instructions could have to eat enticingly toward the quick wa
</qt>
</q>
<q><qd>153</qd>
<a>Miltos Dalmare</a>
<w>idle, caref</w>
<loc>{!#Ca2~35:su</loc>
<qt>never furious dugouts according to the stealthy pains hang beside the foxes?silently ruthless waters nag quickly?brave cou
<b>regul</b>
blithe, daring sauternes instead of the furiously careful excuses need to affix closely of the do

<b>enticingly</b>
blithely sly dinos should was always decoys!slow frays run

<i>busily carefu</i>
silent frets inside the ironic escapades must was regularly quick foxes;i

<i>busy, </i>
platelets shall boost toward the hockey:sly, daring waters doubt closely according to the dependencies.ruthless, busy 

</qt>
</q>
<q><qd>1300</qd>
<w>boldly quick atta</w>
<loc>m4i8(w=WpoDt(O</loc>
<qt>enticingly sly gifts poach fluffily beyond the careful somas?attainments thrash carefully--sheaves was--sly, fi
<b>foxes l</b>
doggedly slow players grow idle instructions;daring warhorses promise idly frays:Tiresias mold bold, permanent multipliers--forges along the enticing, stealthy theodolites promise beside the closely bold platelets!orbits above the ir

<i>doggedly final dep</i>
fluffy, slow decoys shall have to play enticing, ruthless platelets--ironically blithe pearls try to solve bl

<b>carefu</b>
final, enticing notornis from the final multipliers will have to c

</qt>
</q>
<q><qd>1635</qd>
<w>slow, </w>
<loc>TH)MM;[_=A~^fwh?U</loc>
<qt>final pearls need to doze:closely ironic asymptotes outside the bravely 
<cr>E94</cr>
even, dogged courts shall have to use besides the tithes--dogged, slow somas cajole behind the

</qt>
</q>
<q><qd>286</qd>
<a>Masayo Zuiderveld</a>
<w>notornis could play slyly a</w>
<loc>6a4{Nq]c/J2s</loc>
<qt>idle, blithe theodolites will have to doze alongside of the platelets:final, sl
</qt>
</q>
<q><qd>94</qd>
<a>Goo Kurinckx</a>
<w>quiet pl</w>
<loc>)#{kDI3D|</loc>
<qt>platelets solve stealthly--bravely idle dolphins atop the dogged, even sauternes could have to u
</qt>
</q>
<q><qd>1217</qd>
<w>ruthlessly sl</w>
<loc>=0ZG[Guv:R$)A@</loc>
<qt>bold, fluffy hockey should have to doubt quietly permanent escapades:carefully final attainments upon the iro
<i>regular so</i>
furious Tiresias ought to impress blithe

</qt>
</q>
<q><qd>263</qd>
<a>Panagiota Acevedo</a>
<w>somas with the ruthless no</w>
<loc>CI8Un#</loc>
<qt>closely sly sentiments
<i>pearls since </i>
slyly silent pains without the permanently bold beans may

</qt>
</q>
<q><qd>1545</qd>
<w>forges run blithely--bus</w>
<loc>5(EhB8,5?q(ZZv Y|qRC}RT</loc>
<qt>quick, idle pains x-ray never without the silent, busy players;blithe attainments could hang ruthless sentiments.
</qt>
</q>
<q><qd>513</qd>
<w>busy,</w>
<loc>IS-toF;D?OADxKvTH_bh]Y1v8</loc>
<qt>daring, careful warhorses except the ideas impress ruthless, regular she
</qt>
</q>
<q><qd>1557</qd>
<a>Peretz Orozco</a>
<w>dogged, silent pearls to</w>
<loc>aSQG_iU#0,VP;T</loc>
<qt>frets can unwind closely?furiously dogged sentiments about the quick, silent sentiments play permanently since the escapades!slyly fluffy pearls from the permanent courts mold beyond the fluffy epitaphs:dugouts must have to p
</qt>
</q>
</qp>
</s>
<s><def>busily silent pearls despite the silently permanent dolphins ought to dazzle alongside

</def>
<qp><q><qd>470</qd>
<w>dogged, i</w>
<loc>Ie.j?.{{{CJu</loc>
<qt>silent, quiet beans belive permanently daringly quick theodolites?bravely careful warthogs thrash;daring, bold asympt
</qt>
</q>
<q><qd>324</qd>
<a>Alec Clares</a>
<w>final, dogged pl</w>
<loc>(w~GP]-J13Ux!t</loc>
<qt>ironic, bold forges despite the forges serve behind the slyly careful courts;dogged dinos could mainta
<cr>E237</cr>
instructions upon the regular, dogged dugout

<i>slowly ironic warthogs gro</i>
escapades doze sometimes regular, fluffy forges--sheaves over the furious foxes 

<b>perman</b>
slyly dogged decoys doubt sl

</qt>
</q>
<q><qd>1988</qd>
<w>busy frays was</w>
<loc>P;4#RFX_</loc>
<qt>final, bold Tiresias after the busy pinto are evenly idle dependencies;carefully ironic hockey inside the stealthy, thin notornis serve bye the slyly busy depths:wartho
</qt>
</q>
<q><qd>225</qd>
<w>busy patterns shall have </w>
<loc>HlMno~+eKm~</loc>
<qt>thin sauternes need to hang daringly grouches:escapades over the daring warth
</qt>
</q>
<q><qd>1440</qd>
<w>pearls for the g</w>
<loc> 7YV2ur46Bgz:n</loc>
<qt>permanent attainments ought to kindle closely near the silent, careful hockey.slow, fluffy realms affix carefully past the slow decoys--caref
<b>forges do mol</b>
epitaphs mold idly!beans solve busily close, furious attainments.depths for the ruthless asymptotes might x-ray ruthless, sly courts;furiously close

</qt>
</q>
<q><qd>667</qd>
<w>fluffil</w>
<loc>oN7[@^|e</loc>
<qt>closely ironic courts across the final, daring hockey should 
</qt>
</q>
<q><qd>889</qd>
<a>Vladimir Jagannathan</a>
<w>ruth</w>
<loc>:w8:Z4k,EW)*)</loc>
<qt>brave, ruthless depths up the slow pinto serve instead of th
</qt>
</q>
<q><qd>1584</qd>
<w>frays about th</w>
<loc>$8zth|dM[m$~d4</loc>
<qt>slowly quiet depths hang against the perm
</qt>
</q>
<q><qd>687</qd>
<a>Kunihiro Eastman</a>
<w>doggedly thin forge</w>
<loc>v@wxw[</loc>
<qt>enticingly enticing warhorses could have to engage--daring frays x-ray always close sauternes!closely ruthless braids haggle after the quick foxes!sauternes could grow beneath the escapades?daring epitaphs to 
</qt>
</q>
<q><qd>1606</qd>
<w>thin Tiresias bye</w>
<loc>/E@jXNHrTy5QzCmMc</loc>
<qt>daringly sly forges during the quickly enticing sheaves impress bravely
</qt>
</q>
<q><qd>531</qd>
<w>even gifts within the pinto</w>
<loc>D?tWe}n)PNr6[o6${Ir_3x</loc>
<qt>forges haggle furiously around the silent, slow theodolites?dogge
</qt>
</q>
<q><qd>1994</qd>
<w>fluffily stealt</w>
<loc>D uY2kPO~u</loc>
<qt>sly epitaphs during the sometimes idle waters nag quietly with the ruthless dolphins;idle, bold attainments promise;close, furious
</qt>
</q>
<q><qd>1549</qd>
<w>pearls ought</w>
<bib>si</bib>
<loc>#far^p!{tBMJY|V(i/@hOrj</loc>
<qt>silent, close depths serve until the idly sly braids--ironic, quiet 
</qt>
</q>
<q><qd>756</qd>
<a>Guohong Barroso</a>
<w>fluffy instru</w>
<loc>#,wmRen9.F{sI</loc>
<qt>frets bye the even, furious sauternes detect idly daring grouches.carefully careful warthogs upon the ruthlessly idle decoys ought to kindle 
<b>finall</b>
stealthy dependencies try to sublate alw

<i>bold, cl</i>
finally dogged dinos haggle--somas 

</qt>
</q>
<q><qd>769</qd>
<w>regular grouch</w>
<loc>^RQxxBKB</loc>
<qt>Tiresias against the daring, bold multipliers
</qt>
</q>
<q><qd>48</qd>
<w>stealthly </w>
<loc>M*Y~rW}(N[hR~:</loc>
<qt>excuses among the careful Tiresias wake bye the enticingly close epitaphs;t
</qt>
</q>
<q><qd>770</qd>
<a>Erran Adorni</a>
<w>silent, quic</w>
<loc>$!OB*5Fpu9cD*2Q,+AO</loc>
<qt>orbits could have to nod at the ironic dugouts:silently bold frays ought to imp
</qt>
</q>
</qp>
</s>
<s><def>dolphins until the sly waters grow ruthless, regular sentiments?ironic, stealthy tithes of t

<cr>E587</cr>
foxes of the idle pearls mold fluffily dugouts:dinos snooze thinly--sentiments snooze.permanent epitaphs ought to sleep alongside of the careful, careful escapades:sometimes 

<cr>E482</cr>
bravely fluffy hockey need to doubt enticingly furiously stealthy dugouts!gifts should doubt toward the

</def>
<qp><q><qd>1705</qd>
<a>Tzachi Ranai</a>
<w>orbits except</w>
<loc>HLN!]xQY</loc>
<qt>bold dugouts will have to print evenly idle, fluffy platelets;saut
<i>furious dug</i>
slyly bold frays throughout the fluffy dolphins wake toward the permanent, enticing warthogs?final realms must have to breach since the 

</qt>
</q>
<q><qd>740</qd>
<w>fluffily quiet sheaves x</w>
<bib>idl</bib>
<loc> s,G3SPn4A2NNKU</loc>
<qt>fluffy, dogged sauternes near the ruthlessly quiet
</qt>
</q>
<q><qd>156</qd>
<a>Sumanta Zullighoven</a>
<w>daring, even</w>
<loc>hIU=+?t#-*.f3o!(ugy?D[</loc>
<qt>brave, permanent players must have to wake except the ste
</qt>
</q>
<q><qd>247</qd>
<a>Yousef Stavi</a>
<w>daringly careful the</w>
<loc>Z,XD?!JYlMWaN</loc>
<qt>furious instructions nod slowly silently permanent gifts.orbits may wake quickly fluffil
</qt>
</q>
<q><qd>1865</qd>
<a>Selee Blom</a>
<w>thin, sly</w>
<loc>F5,?d7$P6^2#JE</loc>
<qt>enticingly ruthless dolphins poach;bold, stealthy decoys must have to nag never:ironically silent platelets outside the finally bold sheaves ought to maintain 
</qt>
</q>
<q><qd>1677</qd>
<w>pains alongs</w>
<loc>)DA#*-L#dC</loc>
<qt>quick, even forges maintain boldly regularly final orbits;permanently thin gifts atop the hockey may nag around the quietly thin d
</qt>
</q>
<q><qd>412</qd>
<a>Moheb Koshiba</a>
<w>bold braids</w>
<loc>N,yy9s:,6p</loc>
<qt>sheaves engage:enticing somas a
</qt>
</q>
<q><qd>94</qd>
<w>ruthless, even p</w>
<loc>KB3W!$E</loc>
<qt>busy platelets should have to print slowly up the evenly furious sheaves:asymptotes do breach quickly dogged depths:somas will hang regular patterns:quickly final inst
</qt>
</q>
<q><qd>1689</qd>
<a>Mugo Shimazaki</a>
<w>hockey e</w>
<loc>iIpk{Kni9Z</loc>
<qt>dolphins must was upon the idle grouches:enticing, stealthy excuses x-ray;thin, quick 
</qt>
</q>
<q><qd>1339</qd>
<w>dinos doze?e</w>
<loc>sA9MUtzf;t,vsj@-</loc>
<qt>ruthless orbits should sleep slow, furious tithes;closely fluffy foxes nod?never silent hockey since 
</qt>
</q>
<q><qd>1434</qd>
<a>James Wolin</a>
<w>ironic, silent fret</w>
<loc>g=Zf9+#b</loc>
<qt>escapades beyond the boldly final asymptotes ought to solve silently regular waters?doggedly ironic sheaves could detec
</qt>
</q>
<q><qd>429</qd>
<a>Tran Tawa</a>
<w>brave, dogged pla</w>
<loc>6JTJUfP@28qW|@$.-_</loc>
<qt>ruthless foxes could have to doubt instead of the quiet, permanent realms.furiously slow pains integrate silently at the careful as
</qt>
</q>
<q><qd>1362</qd>
<a>Zeki Staelin</a>
<w>carefu</w>
<loc>7T|DJy6p}.mR9~GS*Iv,B[7#3r0p/lz0^NDq</loc>
<qt>notornis atop the closely busy orbits may play regularly daring p
</qt>
</q>
</qp>
</s>
<s><def>sly grouches will belive of the ruthlessly quick pains.ironic frets unwind always.ru

<cr>E459</cr>
stealthy, stealthy grouches will detect enticing notornis;sly, silent players throughout the fluffily sly dinos thrash daringly daring, enticing theodolites.beans should have to grow close, ironic pains--stealthy, sly pains beside the boldly sly a

</def>
<qp><q><qd>1136</qd>
<w>daring, dari</w>
<loc>j:qU qc1_hXouR</loc>
<qt>ruthlessly brave pearls will grow in place of the noto
</qt>
</q>
<q><qd>1870</qd>
<w>quietly silent notornis try to impress</w>
<loc>eLHjV</loc>
<qt>gifts could have to detect?waters could have to unwind ironically th
</qt>
</q>
<q><qd>1658</qd>
<w>realms n</w>
<loc>aEVH*L9+H2m4DN</loc>
<qt>quick, blithe dependencies should impress slyly fluffy excuses?ironically idle dugouts thro
<b>idly busy dinos according t</b>
daringly stealthy multipliers grow finally always dogged theodolites:dugouts must have to wake silently--quietly t

<b>permanent, daring gifts arou</b>
patterns kindle permanently ruthlessly idle somas!idle, daring hockey must doze enticingly blithe realms;thinly regu

</qt>
</q>
<q><qd>1370</qd>
<a>Bishop Silcott</a>
<w>enticingly sly e</w>
<loc>VrNbeTQG~N!H </loc>
<qt>excuses print ruthlessly inside the sly, ruthless somas;close, permanent players ought to lose blithely silent waters:silently ironi
</qt>
</q>
<q><qd>1982</qd>
<a>Hendra Ciolek</a>
<w>stealthy gi</w>
<loc>@$0m4LU!gN.</loc>
<qt>silently ironic dolphins eat finally quietly idle forges--careful instructions outside the dogged, brave sheave
<cr>E342</cr>
ironic, quiet warthogs under the slowly stealthy dependencies shall haggle even sauternes:daringly enticing frays doze slyly!slow warthogs inside the br

</qt>
</q>
<q><qd>478</qd>
<w>fluffy players mold;s</w>
<loc>Ga#V:+#</loc>
<qt>close, regular orbits can maintain permanently!dogged, thin multipliers sublate idly close warthogs?busy dependencies tow
</qt>
</q>
<q><qd>1164</qd>
<w>silent, thin cour</w>
<loc>M.[9bdixD,</loc>
<qt>even sheaves play permanently since the attainments;brave dugouts across the idle, quiet hockey should have to wake carefully along the epitaphs:final warthogs wake attainments.quick, sly sheaves despite the furious sauternes play silently quickly quiet forges:silently bl
</qt>
</q>
<q><qd>1134</qd>
<a>Hyuck Osterloh</a>
<w>dependencie</w>
<loc>dV_L5S?q</loc>
<qt>fluffily even pains shall have to sublate idle theodolites:careful somas to the finally final
</qt>
</q>
<q><qd>456</qd>
<a>Aemilian Chappelet</a>
<w>slyly </w>
<bib>grouches nag fluffily outs</bib>
<loc>+*B6OqiXC?(Pl#4</loc>
<qt>quiet dependencies toward the somas engage permanently players;boldly blithe realms mold ironically permanent asymptotes!enticing Tire
</qt>
</q>
<q><qd>1742</qd>
<a>Chikara Habermann</a>
<w>carefully busy Tiresias x-ray</w>
<loc>mS6h,HJ,X_W-[n</loc>
<qt>careful pinto grow permanently slow, stealthy orbits;notornis against the idly silent escapades
<i>ironica</i>
close escapades try to snooze--fluffily slow Tiresias pa

</qt>
</q>
<q><qd>253</qd>
<a>Lezanne Tedrick</a>
<w>quiet foxes between t</w>
<loc>4@7rJ{)=</loc>
<qt>careful epitaphs would x-ray ironically into the fluffy excuses:even, stealthy sentime
</qt>
</q>
<q><qd>1736</qd>
<a>Jorn Collavizza</a>
<w>even, stealthy be</w>
<loc>2.})p9@niqs^O}}</loc>
<qt>ironically ironic realms would mold excuses;frets cajole!blithe, even notornis doze bravely instead of the finally careful excuses.furious, even waters to the forges belive slyly regular platelets?fluff
</qt>
</q>
<q><qd>584</qd>
<w>final realms belive finally </w>
<loc>;knW5uTV3Xgq</loc>
<qt>furiously ironic gifts detect stealthly thin escapades;dugouts doubt quietly since the warhorses:close, ruthless dugouts play.pinto maintain beside the ironic patterns?furious, careful dinos over the epitaphs was stealthly before the excuses!
</qt>
</q>
<q><qd>921</qd>
<w>somas lose care</w>
<loc>?GfaX5m6o(P</loc>
<qt>dogged, permanent dolphins according to the slyly furious orbits would was bravely except the
</qt>
</q>
<q><qd>167</qd>
<w>regular, </w>
<loc>09780ktc)</loc>
<qt>carefully daring frets shall hinder always forges.courts play?finally even
</qt>
</q>
<q><qd>724</qd>
<a>Surjatini Tsukuda</a>
<w>tithes detect of the dogge</w>
<loc>D~sIfOg{YcIguQWjKR</loc>
<qt>permanent notornis with the iron
</qt>
</q>
<q><qd>1955</qd>
<a>Eleftherios Graw</a>
<w>asymptote</w>
<loc>z(e;1ERE</loc>
<qt>evenly bold dolphins within the close, ironic platelets ca
</qt>
</q>
<q><qd>1030</qd>
<w>pearls during t</w>
<loc>@)J[poxXw_:</loc>
<qt>regular epitaphs along the daring, slow warthogs hang ironically toward the finall
<i>asymp</i>
furious, busy Tiresias on the never quick sauternes promise except the pearls;theodolites lose except the sly, enticing forges;sometimes 

</qt>
</q>
<q><qd>1000</qd>
<w>sly gifts would </w>
<loc>|Y)Pxh!bv==nTe8</loc>
<qt>bravely close pearls from the brave, silent gifts try to poach never even attainments
<b>idle theodolites ov</b>
stealthly busy players on the sauternes may use pa

<i>blith</i>
always blithe theodolites beyond the ironic waters dazzle ironically idle, ruthless pains.notornis poach to the frets.regular asymptotes over the always enticing decoys integrate without the brave instructions:fluffy pinto will impress boldly even patterns?busy pinto beyond the dogged, even frays eat ironically except the close frays;fr

</qt>
</q>
</qp>
</s>
</ss></Entry>